drivers/include/mrf24j40.h

```diff
@@ -115,6 +115,10 @@ extern "C" {
 
 #define MRF24J40_MAX_FRAME_RETRIES      (3U)        /**< Number of frame retries (fixed) */
 
+#ifndef MRF24J40_USE_EXT_PA_LNA
```

Comment:
Add comment about transceiver revisions that support this.

Reply:
Done. See below